notebooks/official/model_evaluation/get_started_with_custom_model_evaluation_import.ipynb

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
<p>nit: you might also provide a link to a calculator to estimate costs of accelerator usage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
added